ss58-registry.json

```
@@ -0,0 +1,275 @@
+			"website": null
+		},
+		{
+			"prefix": "13",
+			"network": "substratee",
+			"displayName": "SubstraTEE",
+			"symbol": null,
+			"decimals": null,
+			"standardAccount": "*25519",
+			"website": null
+		},
+		{
+			"prefix": "16",
+			"network": "kulupu",
+			"displayName": "Kulupu",
+			"symbol": ["KLP"],
+			"decimals": null,
+			"standardAccount": "*25519",
+			"website": null
+		},
+		{
+			"prefix": "18",
+			"network": "darwinia",
+			"displayName": "Darwinia Chain",
+			"symbol": null,
+			"decimals": null,
+			"standardAccount": "*25519",
+			"website": null
+		},
+		{
+			"prefix": "20",
+			"network": "stafi",
+			"displayName": "Stafi",
+			"symbol": null,
+			"decimals": null,
+			"standardAccount": "*25519",
+			"website": null
+		},
+		{
+			"prefix": "23",
+			"network": "shift",
+			"displayName": "ShiftNrg",
+			"symbol": null,
+			"decimals": null,
+			"standardAccount": "*25519",
+			"website": null
+		},
+		{
+			"prefix": "28",
+			"network": "subsocial",
+			"displayName": "Subsocial",
+			"symbol": null,
+			"decimals": null,
+			"standardAccount": "*25519",
+			"website": null
+		},
+		{
+			"prefix": "30",
+			"network": "phala",
+			"displayName": "Phala Network",
+			"symbol": null,
+			"decimals": null,
+			"standardAccount": "*25519",
+			"website": null
+		},
+		{
+			"prefix": "32",
+			"network": "robonomics",
+			"displayName": "Robonomics Network",
+			"symbol": null,
+			"decimals": null,
+			"standardAccount": "*25519",
+			"website": null
+		},
+		{
+			"prefix": "33",
+			"network": "datahighway",
+			"displayName": "DataHighway",
+			"symbol": null,
+			"decimals": null,
+			"standardAccount": "*25519",
+			"website": null
+		},
+		{
+			"prefix": "36",
+			"network": "centrifuge",
+			"displayName": "Centrifuge Chain",
+			"symbol": ["RAD"],
+			"decimals": null,
+			"standardAccount": "*25519",
+			"website": null
+		},
+		{
+			"prefix": "42",
+			"network": "substrate",
+			"displayName": "Substrate",
+			"symbol": null,
+			"decimals": null,
+			"standardAccount": "*25519",
+			"website": "https://substrate.dev/"
+		},
+		{
+			"prefix": "43",
+			"network": "reserved43",
+			"displayName": "This prefix is reserved.",
+			"symbol": null,
+			"decimals": null,
+			"standardAccount": null,
+			"website": null
+		},
+		{
+			"prefix": "44",
+			"network": "chainx",
+			"displayName": "ChainX",
+			"symbol": null,
+			"decimals": null,
+			"standardAccount": "*25519",
+			"website": null
+		},
+		{
+			"prefix": "46",
+			"network": "reserved46",
+			"displayName": "This prefix is reserved.",
+			"symbol": null,
+			"decimals": null,
+			"standardAccount": null,
+			"website": null
+		},
+		{
+			"prefix": "47",
+			"network": "reserved47",
+			"displayName": "This prefix is reserved.",
+			"symbol": null,
+			"decimals": null,
+			"standardAccount": null,
+			"website": null
+		},
+		{
+			"prefix": "48",
+			"network": "reserved48",
+			"displayName": "All prefixes above 48 are reserved.",
```

```suggestion
			"displayName": "All prefixes 48 and higher are reserved and cannot be allocated.",
```

The reason for this change ... at least 2 people have stepped into this trying to claim >48. My wording aims to clarify that, although it is perfectly understandable to me as-is. Then again, my wording may needs some tweaks as well...

---

No, this is definitely better.